research/flamby/fed_isic2019/fenda/run_fold_experiment.slrm

```diff
@@ -81,8 +83,10 @@ source ${VENV_PATH}bin/activate
 echo "Active Environment:"
 which python
 
-for RUN_NAME in "${RUN_NAMES[@]}";
+for ((i=0; i<${#RUN_NAMES[@]}; i++));
 do
+RUN_NAME="${RUN_NAMES[i]}"
```

Comment:
I'm not sure, but does the lack of indentation here cause an issue? I forget if bash scripts are whitespace sensitive (like python). Perhaps worth indenting it to be safe.

Reply:
I don't get error. So it should be fine but I will fix it to make it cleaner.

Reply:
Sounds good. Bash is sometimes finicky and sometimes not with this stuff. Thanks for fixing it for visual purposes anyway.